src/measure_theory/measure/measure_space.lean

```diff
@@ -1776,6 +1776,14 @@ begin
   congr' with x, simp [and_comm]
 end
 
+lemma ae_eq_restrict_of_ae_eq {f g : α → δ} {s : set α} (hfg : f =ᵐ[μ] g) :
+  f =ᵐ[μ.restrict s] g :=
+begin -- note that we cannot use `ae_restrict_iff` since we do not require measurability
+  refine hfg.filter_mono _,
+  rw measure.ae_le_iff_absolutely_continuous,
+  exact measure.absolutely_continuous_of_le measure.restrict_le_self,
+end
```

Comment:
What's `f =ᵐ[μ] g` syntax for? Can you rename the lemma to enable dot notation as `hfg.restrict s`?

Reply:
I've changed the name to `filter.eventually_eq.restrict` so dot notation works now